Shared/Samples/Style symbols from mobile style file/StyleSymbolsFromMobileStyleFileView.swift

```diff
@@ -0,0 +1,190 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct StyleSymbolsFromMobileStyleFileView: View {
+    /// The display scale of the environment.
+    @Environment(\.displayScale) private var displayScale
+    
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// A Boolean value that indicates whether the symbol options sheet is showing.
+    @State private var isShowingSymbolOptionsSheet = false
+    
+    var body: some View {
+        MapView(map: model.map, graphicsOverlays: [model.graphicsOverlay])
+            .onSingleTapGesture { _, mapPoint in
+                // Add the current symbol to map as a graphic at the tap location.
+                let symbolGraphic = Graphic(geometry: mapPoint, symbol: model.currentSymbol?.symbol)
+                model.graphicsOverlay.addGraphic(symbolGraphic)
+            }
+            .toolbar {
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Button("Symbol") {
+                        isShowingSymbolOptionsSheet = true
+                    }
+                    .sheet(isPresented: $isShowingSymbolOptionsSheet, detents: [.large]) {
+                        symbolOptionsList
+                    }
+                    
+                    Spacer()
```

Comment:
Not a big deal but would probably give equal spacing on either side of the spacer.

```suggestion
                    
                    Spacer()
                    
```

or

```suggestion
                    Spacer()
```